client/src/Movies/MovieCard.js

```diff
@@ -1,7 +1,35 @@
 import React from 'react';
+import { Link } from "react-router-dom";
 
 const MovieCard = props => {
-  return;
+  if (!props.movie) {
+      return <div>Loading movie information...</div>;
+    }
+
+    const { title, director, metascore, stars, id } = props.movie;
+    return (
+      <div className="save-wrapper">
+        <div className="movie-card">
+          <Link to="/movies/0" >
```

Comment:
Look out -- this link will take you to the same URL no matter what movie is being displayed. Is that the intended behavior?